4q.txt

```diff
@@ -0,0 +1,26 @@
+import csv
+#CSV FILE
+data = [
+    ["sid", "sname", "salary"],
+    [1, "savan", 3800],
+    [2, "om", 4000],  
+    [3, "sahu", 4500],
+    [4, "hiren", 4100],
+    [5, "sai", 4200]   
+]
+#OPEN A CSV FILE
+with open('salary.csv', 'w', newline='') as f:
+    writer = csv.writer(f)
+    writer.writerows(data)
+s_names = 0
+total_records = 0
+with open('salary.csv', 'r') as f:
+    reader = csv.reader(f)
+    header = next(reader)  
+    for row in reader:
+        total_records += 1
+        sid, sname, salary = row

```

Comment:
explain this statement. what is the purpose?

Reply:
header is variable of  CSV file.
sid, sname, salary is a data rows and  assigning values.
This allows for data processing or analysis while skipping the header row.